### ISSUE TITLE
8.9.x Head has issues with ZSH

### ISSUE THREAD

**Issue by aweingarten** [acquia/blt#1537]
When updating BLT to the lastest 8.9. HEAD I encountered this error.  Assuming it's due to some of the recent alias commits.

```
composer update acquia/blt
Gathering patches for root package.
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 0 installs, 1 update, 0 removals
Gathering patches for root package.
Gathering patches for dependencies. This might take a minute.
  - Updating acquia/blt 8.x-dev (82734f8 => 06f892e):  Checking out 06f892e50b
Writing lock file
Generating autoload files
Updating BLT templated files...
 > blt update
Copying files from BLT's template into your project...
Merging BLT's project.yml template with your project's blt/project.yml...
Removing deprecated files and directories...
> install-alias
[Filesystem\FilesystemStack] touch ["/Users/me/.bash_profile"]
[Filesystem\FilesystemStack]  Failed to touch "/Users/me/.bash_profile".
[Filesystem\FilesystemStack]  Failed to touch "/Users/me/.bash_profile".
[Filesystem\FilesystemStack]  Exit code 1
BLT can automatically create a Bash alias to make it easier to run BLT tasks.
This alias may be created in .bash_profile or .bashrc depending on your system architecture.
```

**Comment by grasmash** [comment]
Does the alias work? Also, is `/Users/me/.bash_profile` writable? Can you try removing the alias and seeing if it gets re-installed?

**Comment by aweingarten** [comment]
If I delete the alias and start with a fresh terminal, and run `composer run-script blt-alias` get `zsh: command not found: blt`

My .bash_profile was not writable.  Making it so looked like this

```
composer run-script blt-alias
> blt install-alias
[Filesystem\FilesystemStack] touch ["/Users/adam.weingarten/.bash_profile"]
BLT can automatically create a Bash alias to make it easier to run BLT tasks.
This alias may be created in .bash_profile or .bashrc depending on your system architecture.
blt
zsh: command not found: blt
```

**Comment by grasmash** [comment]
I don't really know what would cause that error. I made a small modification that may affect things in #1540. Can you try it out?

Also, what's the content of ~/.zsh?

**Comment by aweingarten** [comment]
So I think at this point the issue is that BLT isn't touching zshrc period.

**Comment by grasmash** [comment]
When I run this:

`export SHELL=/bin/zsh; blt install-alias`

BLT does create an alias in ~/zshrc`

**Comment by grasmash** [comment]
Addressed in #1544.
